compiler/luci/import/src/Nodes/CircleBCQFullyConnected.cpp

```diff
@@ -41,6 +41,16 @@ CircleNode *CircleBCQFullyConnectedGraphBuilder::build_node(const circle::Operat
   node->weights_scales(inputs[1]);
   node->weights_binary(inputs[2]);
   node->bias(inputs[3]);
+  node->weights_clusters(inputs[4]);
+
+  // TODO Find and move to appropriate place for setting optional input
+  if (auto bias = dynamic_cast<luci::CircleOutputExclude *>(node->bias()))
+  {
+    // bias is not used for type inference, but node itself should have a type
+    bias->dtype(loco::DataType::FLOAT32);
+
+    // bias is not used for shape inference
```

Comment:
comment seems duplicate

Reply:
It is different with above comment, `type inference` and `shape inference` :)